code/game/objects/structures/catwalk.dm

```diff
@@ -79,6 +79,19 @@
 	if(!QDELETED(src) && severity != 3)
 		physically_destroyed()
 
++/obj/structure/catwalk/grab_attack(var/obj/item/grab/G)
```

Comment:
Stray + at beginning of the line

Reply:
didn't understand what he was doing there